### PR TITLE
MC-9575 Add plugin page for OpenID Connect

### PR DESCRIPTION
**Pull request by pjmonks**
Covers technical aspect of how to use the OpenID Connect plugin, around using endpoints.

### CODE REVIEW

**Review by jamesrwelch — COMMENTED** [review]
I like this a lot.  I won't merge it until the plugin is officially approved / released

File: docs/plugins/openid-connect.md [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+
+The page covering [authentication](../../rest-api/authentication) explains how to authenticate with a basic username/password for a user created directly in Mauro. An alternative authentication method is to use an [OpenID Connect](https://openid.net/connect/faq/) identity service to authenticate users with an external provider/account system, and then authorize them to use Mauro.
+
+In order use OpenID Connect identity providers, the Mauro instance must:
+
+1. Have the [Mauro OpenID Connect Authentication plugin](../../installing/plugins) installed.
+2. Set up and configure one or more identity services that support the OpenID Connect protocol.
+3. Add the configuration details of each identity provider to Mauro.
+
+The full details of this set up can be viewed in the user guide [Using OpenID Connect](../../user-guides/introduction).
```
Comment (olliefreeman) [inline docs/plugins/openid-connect.md:11]: does the relevant pieces of information exist at this page??

**Review by olliefreeman — CHANGES_REQUESTED** [review]
We will want some additional stuff from me regarding configuring details via config to help bootstrap the providers we know about.

Do we have anything yet regarding configuring docker??? (yes i probably should know this but i dont)

File: docs/plugins/openid-connect.md (hunk 1 above)
Comment (pjmonks) [reply docs/plugins/openid-connect.md:11]: The user guide for Open ID connect is pending, notes on what it should include have been forwarded to Gemma (and @jamesrwelch was CC'd in too).

**Pull request merged**